packages/slice-machine/components/Navigation/index.test.tsx

```diff
@@ -128,34 +125,6 @@ describe("Side Navigation", () => {
     expect(element).toBeNull();
   });
 
-  test.each([
```

Comment:
As I said in Slack, I tried with https://www.npmjs.com/package/next-router-mock#example-nextlink-with-react-testing-library and it was still not working.

But I would prefer to skip the test instead of removing it so we can unskip and update it with vite. WDYT?